frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -464,7 +465,7 @@
                 return Date.parse(key.submission__submitted_at_formatted);
             }
             else if (vm.sortColumn === 'rank') {
-                return vm.initial_ranking[key.submission__participant_team__team_name];
+                return vm.initial_ranking[key.id];
```

Comment:
Why are you changing it to `vm.initial_ranking[key.id]`??

Reply:
As there can be more than 1 Baseline `Leaderboard_Data` from Host, so using `id` instead of `participant_team__team_name`